pkg/framework/report.go

```diff
@@ -132,14 +147,27 @@ func parsePodsReview(templatePods []*api.Pod, status Status) []*ClusterCapacityR
 
 	for i := 0; i < templatesCount; i++ {
 		result = append(result, &ClusterCapacityReviewResult{
-			ReplicasOnNodes: make(map[string]int),
+			ReplicasOnNodes: make([]ReplicasOnNode, 0),
 			PodName:         templatePods[i].Name,
 		})
 	}
 
+	replicaCount := make([]map[string]int,templatesCount)
```

Comment:
wondering why still using this map, and not change to ReplicasOnNode? I think that it would also avoid the code below where you are again populating ReplicasOnNode from map[string]int?